DER/DerivativesContracts/Options.rdf

```diff
@@ -370,6 +560,77 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;FutureOptionsExerciseTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ex;AmericanExerciseTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-opt;delivery"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-fi;Future"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">future options exercise terms</rdfs:label>
+		<skos:definition xml:lang="en">Contractual terms defining when or under what circumstances an options instrument may be exercised.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Also includes terms defining delivery when the contract is exercised.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;FxOptionUnderlying">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;Underlier"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+				<owl:someValuesFrom rdf:resource="&fibo-ind-fx-fx;CurrencySpotBuyRate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">fx option underlying</rdfs:label>
+		<skos:definition xml:lang="en">The underlying of the foreign currency option is a currency spot rate.</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;IndexOptionExerciseTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ex;ExerciseTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-opt;delivery"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;AmountOfMoney"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">index option exercise terms</rdfs:label>
+		<skos:definition xml:lang="en">Contractual terms defining when or under what circumstances an options instrument may be exercised.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Also includes terms defining delivery when the contract is exercised. Delivery for index options is by way of cash.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;InterestRateOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionOnFuture"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;InterestRate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">interest rate option</rdfs:label>
+		<skos:definition xml:lang="en">option on some underlying interest rate</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">If the option is exercised, delivery is in the form of the corresponding interest rate future.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
No it's not an option on a Future - see https://www.investopedia.com/terms/i/interestrateoptions.asp